runtime/src/iree/base/string_builder.c

```diff
@@ -248,10 +248,12 @@ IREE_API_EXPORT iree_status_t iree_string_pair_builder_emplace_string(
   if (builder->temp_strings_size == builder->temp_strings_capacity) {
     // Resize.
     builder->temp_strings_capacity = iree_max(8, builder->pairs_capacity * 2);
+    char** realloced = builder->temp_strings;
```

Comment:
ah the capacity too
(otherwise if it fails the next time emplace is called it'll think it has a greater capacity than it does)

Reply:
wait also why is temp_strings_capacity 2* pairs_capacity? I think that should be temp_strings_capacity as well?

Reply:
indeed.